answerking_app/models/serializers.py

```diff
@@ -35,6 +98,7 @@ class ItemSerializer(serializers.ModelSerializer):
     price = serializers.DecimalField(
         max_digits=18,
         decimal_places=2,
+        coerce_to_string=False,
```

Comment:
I think we should set this to false in the settings.base file. That way we ensure subTotal and total are also not sent as strings